source/docs/advanced/payments/payline.md

```diff
@@ -0,0 +1,114 @@
+
+### Install and configure the `Monext_Payline` Magento1 extension
+
+Follow [Monext Payline's documentation](https://docs.monext.fr/display/DT/Plugin+Magento+1) to install and configure the official Magento1 extension. You should configure it in a standard fashion.
+
+### Install the `FrontCommerce_Payline` Magento1 extension
+
+Turn Monext Online (Payline) into a headless payment method by installing [Front-Commerce's `FrontCommerce_Payline` module](https://github.com/front-commerce/magento1-module-payline-front-commerce). No configuration is needed.
+
+```shell
+composer require front-commerce/magento1-module-payline
+```
+
+<blockquote class="note">
+**Note** If your project does not use composer, you can also use [modman](https://github.com/colinmollenhour/modman) or download the latest module version and copy files in your project.
+</blockquote>
+
+### Register the Payline for Magento 1 payment module in Front-Commerce
+
+In your Front-Commerce application:
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/payment-payline-magento1"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+-    { name: "Magento1", path: "server/modules/magento1" }
++    { name: "Magento1", path: "server/modules/magento1" },
++    {
++      name: "Payline",
++      path: "payment-payline-magento1/server/modules/payment-payline",
++    }
+   ],
+   webModules: [
+```
+
+### Register the Payline payment component
+
+1. Override the file that lets you register additional payments forms in Front-Commerce
+```
+mkdir -p my-module/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/
+cp -u node_modules/front-commerce/src/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js my-module/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js
+```
+2. Register Payline
+```diff
++import PaylineMagento1 from "./PaylineMagento1";
+
+const ComponentMap = {
++  PaylineCPT: PaylineMagento1,
+};
+```
+
+### Update your CSPs
+
+To allow loading Monext Online (Payline) related remote resources:
+
+```diff
+// my-module/config/website.js
+  contentSecurityPolicy: {
+    directives: {
+-      scriptSrc: [],
+-      frameSrc: [],
+-      styleSrc: [],
+-      imgSrc: [],
+-      connectSrc: [],
+-      fontSrc: [],
++      scriptSrc: [
++        "*.cdn.payline.com",
++        "pixel.cdnwidget.com",
++      ],
++      frameSrc: ["*.cdn.payline.com"],
++      styleSrc: ["*.cdn.payline.com"],
++      imgSrc: [
++        "*.cdn.payline.com",
++        "*.cdnwidget.com",
++      ],
++      connectSrc: ["*.payline.com"],
++      fontSrc: ["*.cdn.payline.com", "maxcdn.bootstrapcdn.com"],
+      baseUri: []
+    }
+  },
+```
+
+### That's it!
+
+You can now configure the Magento extension to use a HOMO mode and [test the integration](https://docs.monext.fr/display/DT/Les+cartes+de+test)
```

Comment:
`HOMO` mode? It'd be worth mentioning where this configuration is in M1.

Reply:
Yes, good idea. I added a link to https://docs.monext.fr/display/DT/Plugin+Magento+1#PluginMagento1-Commonsettings/Param%C3%A8trescommuns

"HOMO" is the codename for "HOMOLOGATION" but it's named HOMO… while the alternative is "PRODUCTION" :man_shrugging: